sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc

```diff
@@ -61,6 +68,94 @@ function dccr_ftr_workshop_flag_default_flags() {
 }
 
 /**
+ * Implements hook_default_message_type().
+ */
+function dccr_ftr_workshop_default_message_type() {
+  $items = array();
+  $items['full_workshop'] = entity_import('message_type', '{
+    "name" : "full_workshop",
+    "description" : "Full Workshop",
+    "argument_keys" : [],
+    "argument" : [],
+    "category" : "message_type",
+    "data" : {
+      "token options" : { "clear" : 0 },
+      "purge" : { "override" : 0, "enabled" : 0, "quota" : "", "days" : "" }
+    },
+    "language" : "es",
+    "arguments" : null,
+    "message_text" : {
+      "en" : [
+        {
+          "value" : "The workshop [message:field-workshop-node:title] is full",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EThe workshop [message:field-workshop-node:title] is full\\u003C\\/p\\u003E\\n"
+        },
+        {
+          "value" : "The workshop [message:field-workshop-node:title] is full.\\n\\nThanks.",
```

Comment:
It looks bad that the email subject be exactly the same than the email body. 
Could you add something more useful information to the email body?